package.json

```diff
@@ -29,12 +29,14 @@
     "format": "yarn run prettier && yarn run indent"
   },
   "devDependencies": {
+    "ldapjs": "^3.0.2",
     "mocha": "^6.2.1",
     "prettier": "1.16.4",
     "segfault-handler": "^1.3.0"
   },
   "dependencies": {
-    "bindings": "^1.5.0"
+    "bindings": "^1.5.0",
+    "yarn": "^1.22.19"
```

Comment:
I'm not sure if adding yarn here is the best idea. Yarn should really be a global dependency as it a replacement for npm. What would be better still is if i just replaced yarn.lock with package-lock.json and stopped using yarn altogether.